hydra-node/src/Hydra/Chain/Direct/Tx.hs

```diff
@@ -542,21 +542,18 @@ data InitObservation = InitObservation
 -- only returning a Maybe, i.e. 'Either Reason (OnChainTx tx, OnChainHeadState)'
 observeInitTx ::
   NetworkId ->
-  [VerificationKey PaymentKey] ->
```

Comment:
We still need our own cardano `VerificationKey PaymentKey` to determine the asset name we are interested in.